package/libyui-ncurses.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Nov 29 14:41:35 UTC 2019 - Rodion Iafarov <riafarov@suse.com>
+
+- Add support to operate on many widgets with rest-api
```

Comment:
Please mention the so number bump to 11.

Reply:
Added, thanks!